knotx-core/knotx-api/src/main/java/com/cognifide/knotx/api/CustomHandlebarsHelper.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Knot.x - Reactive microservice assembler - API
+ *
+ * Copyright (C) 2016 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.knotx.api;
```

Comment:
I agree that API module may not be the best idea. I think this requires a bit more discussion on how exactly knot.x is used and extended by its users. After we agree on some strategy it seems very reasonably to take it out of the API module.

Speaking about modules - current modules require knot.x applications to depend on all knot.x modules. I think this should also be discussed - ideally implementation modules should not be referenced in applications. Any thoughts?